Makefile

```diff
@@ -18,6 +18,12 @@ VALGRIND ?= valgrind
 GDB		 ?= gdb
 PERF	 ?= perf
 
+# guess clang or gcc (clang sometimes masquerades as gcc because of
+# course it does)
+ifeq ($(shell $(CC) --version | grep clang),clang)
```

Comment:
i don't think this works because the line contains more characters than just `clang`.
```shell
spacetanuki% cc --version | grep clang
Apple clang version 17.0.0 (clang-1700.0.13.3)
spacetanuki% 
```

Reply:
AH, I was trying to avoid the double-negative and thought this was make's findstring. Reverted to `ifneq`. Good catch!

And actually tested with `CC=clang` this time..